Mammoth/TSE/CAccessibilitySettings.cpp

```diff
@@ -0,0 +1,76 @@
+#include "PreComp.h"
+#include "TSEAccessibilitySettings.h"
+#include <sstream>
+
+CAccessibilitySettings::CAccessibilitySettings(void) {
+	m_rgbColorPlayer = CG32bitPixel(255, 255, 255);
```

Comment:
FYI: New C++11/17 style is to initialize this in the class definition instead of here in the constructor.

Reply:
Upcoming commit implements this as well

Reply:
done